p/themes/Origine/origine.css

```
@@ -55,7 +55,8 @@ input:focus, select:focus, textarea:focus {
 	box-shadow: 0 2px 2px #ddf inset;
 }
 
-input:invalid, select:invalid {
+input:focus:invalid, select:focus:invalid,
+input:hover:invalid, select:hover:invalid {
```

> I checked it already: user-invalid works only in Firefox :(

Hence in the future. :-) But only if you include it.

```suggestion
input:user-invalid, select:user-invalid,
input:focus:invalid, select:focus:invalid,
input:hover:invalid, select:hover:invalid {
```